pynetdicom/acse.py

```diff
@@ -422,6 +422,14 @@ def _negotiate_as_requestor(self):
         self.send_request()
         evt.trigger(self.assoc, evt.EVT_REQUESTED, {})
 
+        # Wait for the transport to be ready
```

Comment:
Hah, I see we had the same idea...

But it should be simpler to just put `None` on the `DUL.to_user_queue` if the connection fails.

Reply:
I’ll take a closer look tomorrow but I think if we do this we still have the problem where the acse and connection timeouts run concurrently.